PowerUp/app/src/androidTest/java/powerup/systers/com/powerup/espresso/AvatarActivityTest.java

```diff
@@ -0,0 +1,64 @@
+package powerup.systers.com.powerup.espresso;
+
+import android.content.pm.ActivityInfo;
+import android.support.test.rule.ActivityTestRule;
+import android.support.test.runner.AndroidJUnit4;
+import android.test.suitebuilder.annotation.LargeTest;
+
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import powerup.systers.com.AvatarActivity;
+import powerup.systers.com.R;
+
+import static android.support.test.espresso.Espresso.onView;
+import static android.support.test.espresso.action.ViewActions.click;
+import static android.support.test.espresso.assertion.ViewAssertions.matches;
+import static android.support.test.espresso.matcher.ViewMatchers.isDisplayed;
+import static android.support.test.espresso.matcher.ViewMatchers.withId;
+
+@RunWith(AndroidJUnit4.class)
+@LargeTest
+public class AvatarActivityTest {
+    @Rule
+    public ActivityTestRule<AvatarActivity> activityRule = new ActivityTestRule<>(
+            AvatarActivity.class);
+
+    @Test
+    public void viewTest() {
+        onView(withId(R.id.faceView)).check(matches(isDisplayed()));
+        activityRule.getActivity().setRequestedOrientation(ActivityInfo.SCREEN_ORIENTATION_LANDSCAPE);
```

Comment:
Why are you changing the orientation after you check if the items are displayed and not before?